demos/real_time_stream_analysis/python/real_time_stream_analysis.py

```diff
@@ -0,0 +1,68 @@
+### LICENSE HEADER ###
+
+import argparse
+from stream_reader import StreamReader
+from inference_manager import InferenceManager
+from io_processor import IOProcessor
+from http_visualizer import HttpVisualizer
+from stream_analyzer import StreamAnalyzer
+from logger import LoggerConfig
+from use_cases.person_vehicle_bike_detection import PersonVehicleBikeDetection
+
+
+def get_config():
+	parser = argparse.ArgumentParser(description="Client for real time video stream analysis")
+	parser.add_argument("--stream_url", required=True, type=str, help="URL to the video stream. Example: rtsp://localhost:8888/stream")
+	parser.add_argument("--ovms_url", required=True, type=str, help="Address of OVMS gRPC endpoint. Example: localhost:9000")
+	parser.add_argument("--model_name", required=True, type=str, help="Name of the target model")
+	parser.add_argument("--model_version", required=False, type=int, default=0, help="Version of the taget model. Default: latest available")
+	parser.add_argument("--visualizer_port", required=False, type=int, help="Port of the inferece results visualizer webservice. "
+																  			"If not specified, visualizer will not be launched")
+	parser.add_argument("--binary_input", required=False, action="store_true", help="Convert frames to binary format before sending them to OVMS. Default: False")
```

Comment:
I didn't set `default=False` because I used `action="store_true"`. 
With that if `--binary_input` flag is set in the launching command the value is `True` and if it's not present at all, then it's `False`.